snowfakery/__init__.py

```diff
@@ -10,3 +11,35 @@
 version_file = Path(__file__).parent / "version.txt"
 with version_file.open() as f:
     version = f.read().strip()
+
+
+def generate_data(
+    yaml_file: Union[Path, str],
+    option: List[Tuple[str, str]] = [],
+    dburl: str = None,
+    target_number: Optional[Tuple[int, str]] = None,
+    debug_internals: bool = False,
+    output_format: str = None,
+    output_file: Path = None,
+    output_folder: Path = None,
+    continuation_file: Path = None,
+):
```

Comment:
The GenerateDataFromYaml task currently calls `snowfakery.data_generator.generate` with some arguments that are not included here: `stopping_criteria` and `generate_continuation_file`. How will those be handled when you update CumulusCI to use the new API?

Reply:
stopping_criteria is an internal datatype generated from the target_number. So when CCI stops using Snowfakery internals it will use target_number.

generate_continuation_file was an oversight.

I've also decided to add generate_cci_mapping_file even though only one tool will use it...